content/blog/2020-09-09-september-20-dvc-heartbeat.md

```diff
@@ -0,0 +1,188 @@
+  celebrate 1000 YouTube subscribers!
+picture: 2020-09-09/header.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/september-20-heartbeat/488
+tags:
+  - Heartbeat
+  - CML
+  - DVC
+  - R
+  - Meetup
+  - Videos
+---
+
+Welcome to our September roundup of cool news, new releases, and recommended
+reading!
+
+## News
+
+### Dmitry on Software Engineering Daily
+
+Our CEO Dmitry Petrov was interviewed on the much-beloved Software Engineering
+Daily podcast! Host [Jeff Meyerson](https://twitter.com/the_prion) kicked off
+the discussion:
+
+> Code is version controlled through git, the version control system originally
+> built to manage the Linux codebase. For decades, software has been developed
+> using git for version control. More recently, data engineering has become an
+> unavoidable facet of software development. It is reasonable to ask–why are we
+> not version controlling our data?
+
+For the rest of the episode, listen here!
+
+<external-link
+href="https://softwareengineeringdaily.com/2020/08/24/data-version-control-with-dmitry-petrov/"
+title="Data Version Control with Dmitry Petrov"
+description=""
+link="softwareengineeringdaily.com"
+image="/uploads/images/2020-09-09/sedaily.jpeg"/>
+
+### Contributor's meetup
+
+Last week, we held a meetup for contributors to DVC! Core maintainer
+[Ruslan Kupriev](https://github.com/efiop) hosted a get-together for folks who
+contribute new features, bug fixes, and more to the community. If you missed it,
+you can watch it on YouTube.
+
+https://youtu.be/jUYSTERXxWg
+
+### New videos
+
+We've released several new videos to our growing
+[YouTube channel](https://www.youtube.com/channel/UC37rp97Go-xIX3aNFVHhXfQ)- and
+cool news, we passed 1,000 subscribers!
+
+https://media.giphy.com/media/ZE0JppdERv8t4jVCAt/giphy.gif
+
+_Even Skeletor finds joy in this._
+
+We now have 4 tutorials in our MLOps series. In the latest, we cover how to use
+your own GPU (on-premise or in the cloud) to run GitHub Actions workflows. Check
+it out and give it a try, the code examples are freely available :)
+
+https://youtu.be/rVq-SCNyxVc
+
+We also made our first ever "explainer" video to talk through how DVC works in
+five minutes.
+
+https://youtu.be/UbL7VUpv1Bs
+
+As always, video requests are welcome! Reach out and let us know what topics and
+tutorials you want to see covered. And we appreciate any likes, shares, and
+subscribes on our growing YouTube channel.
+
+## From the community
+
+### A three-part CML series (featuring R!)
+
+DVC ambassador [Marcel Ribeiro-Dantas](https://twitter.com/mribeirodantas) has
+published two of three tutorial blogs in a series on CML! Marcel's use case is
+especially cool because he's using R, plus some causal modeling related to his
+work in bioinformatics, with GitHub Actions.
+
+In Part I, Marcel introduces his project and how he uses DVC, CML and GitHub
+Actions together (with his custom R library).
+
+<external-link
```

Comment:
An issue with rendering the image.
<img width="787" alt="Screen Shot 2020-09-10 at 11 54 44 AM" src="https://user-images.githubusercontent.com/10081867/92785717-801ac400-f35c-11ea-8310-4b0a8001fe1a.png">

Reply:
should be resolved now